tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py

```diff
@@ -152,11 +152,14 @@ def assertTableDoesNotExist(self, fq_table_name):
 
             :param fq_table_name: a fully qualified table name
             """
-            response = self.client.query(
-                f"select count(*) from `{fq_table_name}`")
-            self.assertRaises(gc_exc.GoogleCloudError,
-                              response.result,
-                              timeout=15)
+            stmt = f"select count(*) from `{fq_table_name}`"
+            # response = self.client.query(stmt)
```

Comment:
dead code statement?

Reply:
Yeah, that can be removed.  I'll do a cleanup push.